sorc/chgres_cube.fd/input_data.F90

```diff
@@ -2603,11 +2605,22 @@ subroutine read_input_atm_grib2_file(localpet)
      if (localpet==0) print*, "- LEVEL AFTER SORT = ",slevs(i)
    enddo
  endif
+
+! Is SPFH on full levels Jili Dong
+     do vlev = 1, lev_input
+      iret = grb2_inq(the_file,inv_file,':SPFH:',slevs(vlev))
+      if (iret <= 0) then
+        use_rh = .TRUE.
+        if (localpet == 0) print*,':SPFH:'//slevs(vlev)//' not existing. Use RH instead.'
```

Comment:
Let's update the notification message to be: `':SPFH on level '//trim(slevs(vlev))//' does not exist. Will read in RH and convert to SPFH instead.'` This makes the message shorter (without trim() you get a bunch of empty space) and makes it clear that RH will be read in and later converted to SPFH, not actually used in the output data. I was sort of confused when I first read the output messages, so this change in description might be helpful

Reply:
updated as suggested. Thanks @LarissaReames-NOAA